db/versions/0097.yml

```diff
@@ -0,0 +1,66 @@
+version: 97
+description: Worker pools extra stats
+migrationScript: |-
+  begin
+    create index workers_created_idx on workers (created);
+  end
+downgradeScript: |-
+  begin
+    drop index workers_created_idx;
+  end
+methods:
+  get_worker_pool_error_worker_pools:
+    description: |
+      Returns errors grouped by worker pool
+    mode: read
+    serviceName: worker_manager
+    args: worker_pool_id_in text
+    returns: table (worker_pool text, count integer)
+    body: |-
+      begin
+        RETURN query
+        SELECT worker_pool_errors.worker_pool_id, count(*)::int
+        FROM worker_pool_errors
+        WHERE
+          (worker_pool_id = worker_pool_id_in or worker_pool_id_in is null)
+        GROUP BY worker_pool_errors.worker_pool_id;
+      end
+
+  get_workers_without_provider_data:
+    deprecated: true
+
+  get_worker_manager_workers:
```

Comment:
this replaces `get_workers_without_provider_data` to return workers in the order in which they were created, to get most recent ones on top of the list, instead of ordering them by `worker_id`

This query was used in `workerManager.listWorkersForWorkerGroup()` which was not used  very much at all (only few calls in past 30d), so potentially it is a breaking change for this one call as it will return different workers now.

This endpoint is now used in UI for a separate "Worker manager workers view" page that would only list w-m provisioned workers.